hypha/apply/funds/templates/funds/applicationsubmission_form.html

```diff
@@ -24,7 +24,11 @@ <h2 class="heading heading--no-margin">Editing: {{ object.title }}</h2>
                 {% endif %}
             {% endfor %}
             {% for button_name, button_type, button_value in buttons %}
-                <input class="button button--submit button--top-space button--{{ button_type }}" type="submit" name="{{ button_name }}" value="{{ button_value }}" />
+                {% if button_value == 'Save Draft' %}
+                    <input class="button button--submit button--top-space button--{{ button_type }}" type="submit" name="{{ button_name }}" value="{{ button_value }}" formnovalidate/>
+                {% else %}
+                    <input class="button button--submit button--top-space button--{{ button_type }}" type="submit" name="{{ button_name }}" value="{{ button_value }}" />
+                {% endif %}
```

Comment:
Can't we do a simple `{% if button_value == 'Save Draft' %} formnovalidate{% endif %}`, less code.